src/test/resources/de/numcodex/sq2cql/testCases/1-age.json

```diff
@@ -17,7 +17,7 @@
                         "code": "a",
                         "display": "years"
                     },
-                    "value": 16
+                    "value": 19
```

Comment:
Why is that change necessary?

Reply:
Because prior the age was requested via the extension defined in the codex test data. Now we use the age calculated by date which is 19. i assume that he was 16 at the date of creating in the test instance. Which underlines the use of the new methodology.

Reply:
Ok I hope the test doesn't depend on the execution date.

Reply:
Fair point changed the query to use greater equal.